lib/module.js

```diff
@@ -39,6 +39,7 @@ function httpModule(_moduleOptions) {
     proxyHeadersIgnore: ['accept', 'host', 'cf-ray', 'cf-connecting-ip'],
     proxy: false,
     retry: false,
+    timeout: 10000,
```

Comment:
I think we can default it to `false` to be same behaviour of axios. Making migration easier. Also can have `serverTimeout` and `clientTimeout` options. Lower timeout makes sense for SSR.